_posts/11-02-01-Test-Driven-Development.md

```diff
@@ -1,40 +1,42 @@
 ---
 isChild: true
-title: Testiranje zasnovano na razvoju
-anchor: test_driven_development
+anchor:  test_driven_development
 ---
 
-## Testiranje zasnovano na razvoju {#test_driven_development_title}
+## Razvoj vođen testiranjem (Test Driven Development - TDD) {#test_driven_development_title}
 
-Sa [Wikipedia](http://en.wikipedia.org/wiki/Test-driven_development):
+Sa [Wikipedia-je](http://en.wikipedia.org/wiki/Test-driven_development):
 
-> Testiranje zasnovano na razvoju (TDD) je proces razvoja softvera koji se oslanja na ponavljanje kratkih razvojnih
-krugova: prvo se piše slučaj neispravnog automatizovanog testa koji definiše željeno poboljšanje ili novu funkciju, onda
-pravi kod koji će proći test i na kraju refaktoriše novi kod po prihvatljivim standardima. Kent Beck, koji je zaslužan
-za razvoj ili ponovno otkrivanje tehnike, izjavio je 2003 da TDD podstiče jednostavan dizajn i uliva poverenje.
+> Razvoj vođen testiranjem (TDD) je proces razvoja softvera koji se bazira na ponavljanju kratkih razvojnih
+> krugova: programer najpre piše neispravan automatizovan testni slučaj koji definiše željeno poboljšanje ili
+> novu funkcionalnost, a onda piše kôd koji će zadovoljiti test i na kraju refaktoriše novi kôd u skladu sa standardima.
+> Kent Beck, koji je zaslužan za razvoj ili 'ponovno otkrivanje' ove tehnike, izjavio je 2003. godine da TDD
+> podstiče jednostavan dizajn i podiže samopouzdanje.
 
-Postoji nekoliko različitih tipova testiranja koje možete koristiti za testiranje vaše aplikacije
+Postoji nekoliko različitih tipova testiranja koje možete koristiti za testiranje vaše aplikacije:
 
-### Jedinično testiranje
+### Unit testiranje
 
-Jedinično testiranje je programski pristup koji treba da obezbedi da funkcije, klase i metode rade onako kako smo
-očekivali, sa stanovišta načina kako smo ih napravili kroz ceo razvojni krug. Proverom ulaznih i izlaznih vrednosti
-raznovrsnih funkcija i metoda, možete proveriti da li je interna logika ispravna. Korišćenjem zavisnog ubrizgavanja
-(Dependency Injection) i pravljenjem "lažnih" klasa i stubova možete proveriti da li se zavisnosti ispravno koriste radi
-još bolje pokrivenosti testa.
+Unit testiranje je pristup koji obezbeđuje da funkcije, klase i metode rade onako kako se očekuje, od momenta
```

Comment:
Nisam siguran koliko je adekvatno reći "programski" pristup. U originalu stoji: "Unit Testing is a programming approach...". Onda bi pre bilo nešto tipa "programerski pristup" ili "pristup u programiranju". Ali sve mi to nekako deluje konfuzno i prenatrpano, i baš iz tog razloga sam izbacio tu reč, jer mislim da nije neophodna.